pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -228,6 +248,25 @@ buildStdenv.mkDerivation ({
     # RBox WASM Sandboxing
     export WASM_CC=${pkgsCross.wasi32.stdenv.cc}/bin/${pkgsCross.wasi32.stdenv.cc.targetPrefix}cc
     export WASM_CXX=${pkgsCross.wasi32.stdenv.cc}/bin/${pkgsCross.wasi32.stdenv.cc.targetPrefix}c++
+ '') + (lib.optionalString pgoSupport ''
```

Comment:
Why are all those lib.optionalString in `()`? Is there some kind of nested condition here I am missing?

Reply:
Indeed these are not required, removed.